reference/kubectl-plugin.md

```diff
@@ -36,24 +27,36 @@ kubectl-plugin 1.0.0
 
 ### Using Mayastor kubectl plugin
 
+
 ```
 USAGE:
     kubectl-mayastor [OPTIONS] <SUBCOMMAND>
 
-FLAGS:
-    -h, --help       Prints help information
-    -V, --version    Prints version information
-
 OPTIONS:
-    -j, --jaeger <jaeger>      Trace rest requests to the Jaeger endpoint agent
-    -o, --output <output>      The Output, viz yaml, json [default: none]  [possible values: yaml, json, none]
-    -r, --rest <rest>          The rest endpoint, parsed from KUBECONFIG, if left empty 
-    -t, --timeout <timeout>    Timeout for the REST operations [default: 10s]
-
+    -h, --help
+            Print help information
+    -j, --jaeger <JAEGER>
+            Trace rest requests to the Jaeger endpoint agent
+    -k, --kube-config-path <KUBE_CONFIG_PATH>
+            Path to kubeconfig file
+    -n, --namespace <NAMESPACE>
+            Kubernetes namespace of mayastor service, defaults to mayastor [default: mayastor]
+    -o, --output <OUTPUT>
+            The Output, viz yaml, json [default: none]
+    -r, --rest <REST>
+            The rest endpoint to connect to
+    -t, --timeout <TIMEOUT>
+            Timeout for the REST operations [default: 10s]
+    -V, --version
+            Print version information
 SUBCOMMANDS:
-    get      'Get' resources
-    help     Prints this message or the help of the given subcommand(s)
-    scale    'Scale' resources
+    cordon      'Cordon' resources
+    drain       'Drain' resources
```

Comment:
Drain functionality may or may not make it to the release. In case, if it doesn't, we need a PR to remove this help section. Similar changes needed in the code as well.
FYI @chriswldenyer @sinhaashish @Abhinandan-Purkait